### PR TITLE
Update UG and DG: Change Add Command to Insert Command

### LINKED ISSUE #163: Change Add command field n/NAME to n/CUSTOMER_NAME

### LINKED ISSUE #165: Change Add Command to Insert Command

### PR DESCRIPTION
**Pull request by Amoscheong97**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -812,18 +812,20 @@ These instructions only provide a starting point for testers to work on; testers
...
-   Expected: The order is added to the list, including the item type and the order comment
+.. Test case: `insert` `tid/9876543210` `n/John Doe` `a/Blk 572 Hougang st 51 #10-33 S530572` `p/98766789` `ts/2020-02-20 1300` `w/Yishun` `pm/Cash` +
+   Expected: Inserts an order with the above details to the list and displayed on the GUI
+.. Test case: `insert` `tid/1023456789` `n/Amos Cheong` `a/Blk 572 Hougang st 51 #11-37 S530572` `p/9001 0019` `ts/2020-03-10 1650` `w/Marsiling` `pm/Nets` `c/Leave it at the riser` `type/glass`
+   Expected: Inserts the order to the list, including the item type and the order comment
 .. Test case: Invalid Syntax
-   Expected: No order is added. Error details shown in the response message. A help message displayed for user to add accordingly. Status bar remain unchanged
+   Expected: No order is added. Error details shown in the response message. A help message displayed for user to insert accordingly. Status bar remain unchanged
+.. Test case: Insert order with existing Transaction ID in list
+   Expected: An error will occur and a message will be displayed, stating that order with duplicate ID cannot be inserted into the list
```
Comment (JeremyLoh) [inline docs/DeveloperGuide.adoc]: There should be a newline for 'Expected' section in DeveloperGuide's G2: Inserting an order 
(Appendix G: Instructions for Manual Testing)
image

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -177,7 +177,7 @@ Step 2. The user executes `delete 5` command to delete the 5th person in the add
 
 image::UndoRedoState1.png[]
 
-Step 3. The user executes `add n/David ...` to add a new person. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `insert n/David ...` to add a new person. The `insert` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: > Step 3. The user executes `insert n/David ...` to add a new person.
Perhaps this sentence, can change from to "to add a new person." to "to add a new delivery orders."

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -812,18 +812,20 @@ These instructions only provide a starting point for testers to work on; testers
...
    Expected: Shows the GUI of the Delino App. The window size may not be optimum
 
-=== Adding an order
+=== Inserting an order
 
-. Add a minimum of 2 orders
+. Insert a minimum of 2 orders
 
-.. Add command format: `add` `tid/TRANSACTION_ID` `n/NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
+.. Insert command format: `insert` `tid/TRANSACTION_ID` `n/CUSTOMER_NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: pm/PAYMENT_METHOD - I think we have discussed that this should change to CASH_ON_DELIVERY, sorry I did not state it down in the telegram message.
Perhaps the prefix can change to cod. If you do decide to change please also change it in the prefix appendix! and refactor it accordingly, Thank you!

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (khsc96) [reply docs/DeveloperGuide.adoc]: this includes all the to add new person or adding person should be changed to inserting order delivery details or things along that line.

> Step 4. The user now decides that adding the person was a mistake, and decides to undo that action by executing the `undo` command. The `undo` command will call `Model#undoAddressBook()`, which will shift the `currentStatePointer` once to the left, pointing it to the previous address book state, and restores the address book to that state.

If possible can change whatever you can spot if not when we do a review standardising in later stage we can keep a look out on such details

**Review by khsc96 — COMMENTED** [review]
LGTM, minor documentation changes if possible please do make the appropriate changes. We can do a team review at the end of milestone v1.2 to look out for such mistakes.

**Review by JeremyLoh — APPROVED** [review]
Looks good to merge.

**Review by khsc96 — APPROVED** [review]
LGTM

**Pull request merged**